src/tests/JIT/Regression/JitBlue/Runtime_92201/Runtime_92201.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Optimize>True</Optimize>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="$(MSBuildProjectName).cs" />
+
+    <CLRTestEnvironmentVariable Include="DOTNET_TieredCompilation" Value="0" />
+    <CLRTestEnvironmentVariable Include="DOTNET_EnableHWIntrinsic" Value="0" />
+    <CLRTestEnvironmentVariable Include="DOTNET_JitStress" Value="9a" />
```

Comment:
Tests should  specify the exact right stress modes via `DOTNET_JitStressModeNames` as the exact set of enabled stress modes can otherwise end up changing in the future due to other changes. Also if possible it would be nice to reduce the test case to something minimal

Reply:
It would take some time to produce a minimal test and don't want to spend that much more time on it, but I can put in the stress mode names

Reply:
I cleaned some the test up.